index.js

```diff
@@ -184,7 +184,7 @@ async function linkModule(moduleName) {
         }
       }
     }
-    await symlink(path.join('../', target), moduleDir, DIR_LINK_TYPE);
```

Comment:
Would need to understand the change better in order to approve ...

Reply:
See below.